README.md

```diff
@@ -1,70 +1,100 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+## Air Pollution Data App
+
+This is a Single Page Application designed for mobile phones. It is created with React & Redux and uses 2 different API's to store a list of countries, cities and air pollution data for a given city. It has a search bar where you can directly look up for air pollution data of a city.
+
+
+## Built With
+
+- Editor : Visual studio code
+
+- Languages
+  - [x] HTML
+  - [x] CSS
+  - [x] JavaScript
+
+- Libraries
+  - [x] React
+  - [x] Redux
+
+## Live Demo Link
+
+[Live Demo](https://dodoburner.github.io/europe-air-pollution/)
```

Comment:
- [ ] Great job with the video presentation 📸 👍🏻  however as requested by the previous reviewer, make sure the project is deployed properly, and the home and detail page works as expected. Your deployed link in the README directs me to an empty page. Kindly make sure the project is deployed properly using Heroku, Netlify, or GitHub Pages. 👍🏻 💯